core/payment-driver/base/src/utils.rs

```diff
@@ -0,0 +1,35 @@
+/*
+    Common utility functions for dealing with PaymentDriver related objects
+*/
+
+// External crates
+use chrono::Utc;
+use serde_json;
+
+// Workspace uses
+use ya_core_model::driver::{GenericError, PaymentConfirmation, PaymentDetails, SchedulePayment};
+
+pub fn to_payment_details(msg: SchedulePayment) -> PaymentDetails {
+    PaymentDetails {
+        recipient: msg.recipient().to_string(),
+        sender: msg.sender().to_string(),
+        amount: msg.amount(),
+        date: Some(Utc::now()),
+    }
+}
+
+// Used by the DummyDriver to have a 2 way conversion between details & confirmation
+pub fn to_confirmation(details: &PaymentDetails) -> Result<Vec<u8>, GenericError> {
```

Comment:
This is totally dummy-driver-specific. Do we really want it here? Other drivers will rather use transaction hash or ID as confirmation.

Reply:
not really, looked too late at GNT driver to find out this was dummy stuff..